sources/TileDB.CSharp/Enums.cs

```diff
@@ -18,11 +18,13 @@ public enum ObjectType : uint
         TILEDB_ARRAY = tiledb_object_t.TILEDB_ARRAY
     }
 
-
     public enum QueryType : uint
     {
         TILEDB_READ = tiledb_query_type_t.TILEDB_READ,
         TILEDB_WRITE = tiledb_query_type_t.TILEDB_WRITE,
+        Delete = tiledb_query_type_t.TILEDB_DELETE,
```

Comment:
I gave C#-friendly names to the new enum values, will update the others soon (and obsolete the `TILEDB_***` values).